lib/settings_base.py

```diff
@@ -1477,3 +1477,7 @@ def read_only_mode(env):
         'rest_framework.renderers.JSONRenderer',
     ),
 }
+
+# This is the DSN to the local Sentry service. It might be overidden in
+# site-specific settings files as well.
+SENTRY_DSN = os.environ.get('SENTRY_DSN')
```

Comment:
I don't think this is configured in docker yet, is it? Do we already have a local sentry service?